documentation/docs/guides/share-goose-sessions.md

```diff
@@ -1,16 +1,37 @@
 ---
 sidebar_position: 5
-title: Making Shared Goose Agents
-sidebar_label: Share Sessions
+title: Sharing a Goose Agent
+sidebar_label: Share Goose Agents
 ---
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 
-A shared session in Goose is like a collaborative workspace where multiple people can work with the AI assistant together in real-time. Think of it similar to a shared Google Doc, but for AI assistance.
+A shared Agent in Goose is like a collaborative workspace where multiple people can work with the AI assistant together in real-time. Think of it similar to a shared Google Doc, but for AI assistance.
```

Comment:
I'd say switching more to a `template` as an example pending clarification on the shared history.